frontend/src/pages/ChainEdit.js

```diff
@@ -149,33 +149,83 @@ const ChainEdit = () => {
-                        onChange={(e) => handleChange(e, genders, setGenders)}
-                        checked={genders[value] ? true : false}
-                      />
-                    );
-                  })}
+                  <Typography
+                    component="p"
+                    style={{ textAlign: "left", textTransform: "capitalize" }}
+                  >
+                    categories
+                  </Typography>
+                  <div style={{ display: "flex", padding: "2% 0" }}>
+                    {categories.genders.map((value, i) => {
+                      return (
+                        <div key={value}>
+                          <input
+                            className="map-cat-input"
+                            key={`input-${value}-${i}`}
+                            id={`${value}`}
+                            type="checkbox"
+                            name={value}
+                            onChange={(e) =>
+                              handleChange(e, genders, setGenders)
+                            }
+                            checked={genders[value] ? true : false}
+                          ></input>
+                          <label
+                            key={`label-${value}-${i}`}
+                            htmlFor={`${value}`}
+                          >
+                            <Typography variant="body2">{`${value}'s clothing`}</Typography>
+                          </label>
+                        </div>
+                      );
+                    })}
+                  </div>
                 </div>
 
                 <div>
-                  <Typography component="p">sizes</Typography>
-                  {categories.sizes.map((value) => {
-                    return (
-                      <CheckboxField
-                        name={value}
-                        label={value}
-                        onChange={(e) => handleChange(e, sizes, setSizes)}
-                        checked={sizes[value] ? true : false}
-                      />
-                    );
-                  })}
+                  <Typography
+                    component="p"
+                    style={{ textAlign: "left", textTransform: "capitalize" }}
+                  >
+                    sizes
+                  </Typography>
+                  <div style={{ display: "flex", padding: "2% 0" }}>
+                    {categories.sizes.map((value, i) => {
+                      return (
+                        <div key={value}>
+                          <input
+                            className="map-cat-input"
+                            key={`input-${value}-${i}`}
+                            id={`${value}`}
+                            type="checkbox"
+                            name={value}
+                            onChange={(e) => handleChange(e, sizes, setSizes)}
+                            checked={sizes[value] ? true : false}
```

Comment:
Isn't this just checked={sizes[value]}

sizes is already a boolean array. Otherwise try checked={!!sizes[value]} or checked={Boolean(sizes[value])}(https://www.samanthaming.com/tidbits/19-2-ways-to-convert-to-boolean/).